test/Microsoft.ApplicationInsights.AspNetCore.Tests/JavaScript/ApplicationInsightsJavaScriptTest.cs

```diff
@@ -11,19 +13,21 @@
 
     public static class ApplicationInsightsJavaScriptTest
     {
+        private static JavaScriptEncoder encoder = JavaScriptEncoder.Create(new UnicodeRange[] { UnicodeRanges.BasicLatin });
```

Comment:
is it what Chinese or Russian customers will expect? Which layer will decode it back to localized string?